apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -123,7 +123,7 @@ function disable_coming_soon_on_privacy_change( $old_value, $value ) {
  */
 function add_option_to_new_site( $blog_id, $user_id, $domain, $path, $network_id, $meta ) {
 	if ( 0 === $meta['public'] && 1 === (int) $meta['options']['wpcom_public_coming_soon'] ) {
-		add_blog_option( $blog_id, 'wpcom_public_coming_soon', 1 );
+		update_option( 'wpcom_public_coming_soon', 1 );
```

Comment:
Is this change safe? From looking at where the `wpmu_new_blog` action is fired in `ms-site.php`, it looks like the site hasn't been switched to, so I'm wondering if this will actually try to change the option for the API rather than the desired site. I might have missed something, though!

Reply:
Just tested and confirmed that when called in this function, `get_current_blog_id()` returns a different `id` to `$blog_id` so this change will need to be reverted.

Reply:
I've reverted this to use `add_blog_option` and wrapped it in a `function_exists` check just to be (ultra) careful. It's not _really_ needed because the `wpmu_new_blog` hook will never be called in a non multi-site environment, but since the plugin _is_ deployed in single site environments, the extra check feels nice to me. Happy to remove it if anyone prefers, though!

Reply:
Since we're using `add_blog_option`, though, to get the tests working, I've updated the `phpunit.xml.dist` file to set the `WP_TESTS_MULTISITE` constant so that the tests are run in an multi site environment where they have access to this function.